custom_components/smartthinq_sensors/sensor.py

```diff
@@ -168,7 +168,7 @@ class ThinQSensorEntityDescription(SensorEntityDescription):
     ),
     ThinQSensorEntityDescription(
         key=FEAT_TUBCLEAN_COUNT,
-        name="Tube clean counter",
+        name="Tub clean counter",
```

Comment:
Problem here is that entity ID will remain `tube_clean_counter`, and if I change this will break possible automation based on this. Wondering if make sense change anyway only the description...

Reply:
Fair point, what's your opinion @philbion?

Reply:
Cheers, thanks for your time and fix :)

Reply:
Not really sure that entity ID will not change, because unique ID do not change. 
May be we can go ahead and see what happen, in case it change I will put a note about "Breaking change" in release note and i will receive 10-20 ticket stating that `tube_clean` count sensors is not available anymore (because nobody read release note 🤐).

Reply:
It's up to you; You can always update the issue template and note about that breaking change.